packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -283,14 +286,12 @@ export async function generateImageData({
     const transforms = imageSizes.sizes.map(outputWidth => {
       const width = Math.round(outputWidth)
       const transform = createTransformObject({
-        quality,
-        ...transformOptions,
-        fit,
-        cropFocus,
+        ...sharedOptions,
         ...args.webpOptions,
         width,
         height: Math.round(width / imageSizes.aspectRatio),
         toFormat: `webp`,
+        background: backgroundColor,
```

Comment:
Should we passing this if we already have sharedOptions? That seems redundant.

Reply:
Oops. Left that in after I refactored the others into sharedOptions